config/rules/password-secret.yaml

```diff
@@ -572,7 +572,7 @@ rules:
       - CWE-257
       - CWE-259
   - Code: 3065
-    Pattern: heroku.*[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}
+    Pattern: heroku.{0,55}[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}
```

Comment:
can we add some unit test for the same. https://github.com/americanexpress/earlybird/tree/main/pkg/scan